libs/design/src/molecules/card/card-media/card-media.directive.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  Directive,
+  HostBinding,
+} from '@angular/core';
+
+@Directive({
+  selector: '[daffCardMedia]',
```

Comment:
I don't think this makes sense. I don't know of any use-case for supporting rendering a video inside a card. Seems likely that the UI (on a mobile device) is much too small. I think it makes more sense to keep Image until we have an explicit use-case for media.

Reply:
Reverted.